repository-meta-analyzer/src/test/java/org/acme/repositories/NugetMetaAnalyzerTest.java

```diff
@@ -30,14 +31,22 @@
 import org.mockserver.client.MockServerClient;
 import org.mockserver.integration.ClientAndServer;
 
+import javax.inject.Inject;
 import java.nio.file.Files;
 import java.nio.file.Paths;
 
 import static org.mockserver.model.HttpRequest.request;
 import static org.mockserver.model.HttpResponse.response;
 
+@QuarkusTest
 class NugetMetaAnalyzerTest {
 
+    @Inject
+    NugetMetaAnalyzer analyzer;
+
+    @Inject
+    Component component;
```

Comment:
This should not be necessary. `Component` is not supposed to be a CDI-managed bean.